src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2486,15 +2486,16 @@ export class StudyViewPageStore {
             };
         }
 
-        if (!_.isEmpty(this.mutationProfiles.result)) {
-            _chartMetaSet[UniqueKey.FUSION_GENES_TABLE] = {
-                uniqueKey: UniqueKey.FUSION_GENES_TABLE,
-                dataType: getChartMetaDataType(UniqueKey.FUSION_GENES_TABLE),
-                patientAttribute:false,
-                displayName: 'Fusion Genes',
-                priority: getDefaultPriorityByUniqueKey(UniqueKey.FUSION_GENES_TABLE),
-                renderWhenDataChange: true,
-                description: ''
+        if ((!_.isEmpty(this.molecularProfileSampleCounts.result)) &&
```

Comment:
I don't think this condition is needed anymore

Reply:
webstorm will throw you an error if you remove the first nullity check.